docs/projects/techolympics25.md

```diff
@@ -0,0 +1,185 @@
+# TechOlympics 2025
+
+This guide is a writeup of the the "Norse IoT Websocket Workshop" session at TechOlympics 2025.
+
+Here is the [presentation](https://docs.google.com/presentation/d/1rqTwmdk6xH_lqwWt_yxc8ZMDEDefVV2FJRK6BXMFeCU/edit#slide=id.g330fa5e991b_0_15) for the session. These docs go over the same content as the slide deck.
+
+## So, what is IoT?
+
+The Internet of Things (IoT) describes the network of physical objects — "things" — that are embedded with sensors, software, and other technologies for the purpose of connecting and exchanging data with other devices and systems over the internet.
+
+IoT Devices range from ordinary household objects to sophisticated industrial tools. 
+
+Oracle estimates there will be 10 billion connected IoT devices today; 22 billion by 2025.
+
+## What are we using?
+
+We're using ESP-32 devices, which strike a good balance between power and cost effiency. ESP-32s are a powerful, lightweight chip, anda re useful for many scenarios. They're Wi-Fi/Bluetooth/Matter enabled allowing for various projects.
+
+## What are we building?
+
+- A Wi-Fi Enabled Game Controller
+- Will feature four buttons to send responses and a buzzer to communicate right/wrong.
+- Will send data to an open-source Kahoot clone through Websockets.
+- Will receive data (correct/incorrect) and make a noise.
+
+
+## The Code
+
+Participants had the code pre-downloaded onto their ESP-32s. If you're looking to follow along at home, you'll need to clone our [Rahoot repo](https://github.com/Norse-IoT/Rahoot), and use Arduino IDE to image your ESP-32 with the [client code](https://github.com/Norse-IoT/Rahoot/tree/main/client/KahootClient). For more info, try out our [Getting Started](/getting-started) guide.
+
+You'll also need a webserver running our fork of Rahoot. Follow the repo's [Getting Started](https://github.com/Norse-IoT/Rahoot?tab=readme-ov-file#-getting-started) guide for more information. The Rahoot server receives data over Websockets and displays it on a website.
+
+
+## The Hardware
+
+- 1x [ESP-32](https://www.amazon.com/dp/B0718T232Z/)
+- 1x [Breadboard](https://www.amazon.com/dp/B07LFD4LT6/)
+- 1x [Buzzer](https://www.amazon.com/dp/B07VK1GJ9X)
```

Comment:
Note that this is the wrong link. These are active buzzers, and they _must_ use passive buzzers.

Reply:
This would work: https://www.amazon.com/dp/B01NCOXB2Q

This is what I got: https://www.amazon.com/dp/B07GBTJP9F